docs/articles/baseplatform/creating-sdk.rst

```diff
@@ -5,8 +5,8 @@
 Building the SDK installer
 ===========================
 
-Building the SDK installer is not very different from creating the corresponding
-target image. All you have to do, is pass an additional option to bitbake:
+Building the SDK installer is not very different from the instructions on `building-pelux sources <http://pelux.io/software-factory/master/chapters/baseplatform/building-PELUX-sources.html>`_ to create the corresponding target image. All you have to do, is pass an additional option to bitbake:
```

Comment:
is not very different from PELUX from sources or so

Reply:
Can you elaborate more what do you mean?